integration-cli/docker_cli_swarm_test.go

```diff
@@ -130,13 +130,15 @@ func (s *DockerSwarmSuite) TestSwarmInit(c *testing.T) {
 }
 
 func (s *DockerSwarmSuite) TestSwarmInitIPv6(c *testing.T) {
-	testRequires(c, IPv6)
```

Comment:
Wouldn't that test fail on machines where IPv6 is disabled?

Reply:
The now-removed `IPv6` function was broken, it'd return false if IPv6 was disabled.

Apparently we don't have any test runners with IPv6 disabled, and we have many other tests that assume it'll work (in the CLI suites, and in networking tests). The two tests modified in this PR hadn't been running anywhere.

So, I didn't try to replace or fix `IPv6` ... if we do start testing on kernels without IPv6 support, there will be a lot of tests that need to be skipped.